web/satellite/src/components/team/ProjectMembersArea.vue

```diff
@@ -57,12 +60,15 @@
 
 <script lang="ts">
     import { Component, Vue } from 'vue-property-decorator';
-    import TeamMemberItem from '@/components/team/TeamMemberItem.vue';
-    import HeaderArea from '@/components/team/headerArea/HeaderArea.vue';
-    import Footer from '@/components/team/footerArea/Footer.vue';
+    import HeaderArea from '@/components/team/HeaderArea.vue';
+    import List from '@/components/common/List.vue';
     import { NOTIFICATION_ACTIONS, PM_ACTIONS } from '@/utils/constants/actionNames';
-    import { TeamMember } from '@/types/teamMembers';
+    import Pagination from '@/components/common/Pagination.vue';
+    import { ProjectMember, ProjectMemberOrderBy, ProjectMembersPage } from '@/types/projectMembers';
+    import ProjectMemberListItem from '@/components/team/ProjectMemberListItem.vue';
     import { RequestResponse } from '@/types/response';
+    import { SortDirection } from '@/types/common';
```

Comment:
disordered

Reply:
This is correct imports sort order. Alphabetical